saleor/graphql/account/types.py

```diff
@@ -67,7 +105,7 @@ class Meta:
         model = get_user_model()
 
     def resolve_addresses(self, info, **kwargs):
-        return self.addresses.all()
+        return self.get_addresses()
```

Comment:
Here we could use `self.addresses.annotate_default(self)`, couldn't we? And then we won't need `get_addresses` function anymore.

Reply:
Yeah we could, I'm not sure which one is better, exposing ```get_addresses``` in model level or just accessing queryset manager in graphql API level.

Maybe let's keep it simple, without so many abstract layers.

Reply:
Resolvers/views are a piece of code where you usually fetch necessary data from the DB so using querysets there is totally fine in my opinion. Also, querysets are a clear and readable method of encapsulating the logic that you want to perform on database tables. So I think we ended up with a very nice implementation 👍

Reply:
That's true. Thanks for the detailed review.